config.yaml

```diff
@@ -33,7 +36,7 @@ output:
   - Self Diffusion Coefficient Array
   - Specific Heat Capacity
   - Lindemann criterion
-
+out_dir: 
```

Comment:
Maybe undefined to have empty out_dir like this. Specify a directory or remove it.

Reply:
Fixed!